network/common/src/main/java/org/apache/spark/network/server/MessageHandler.java

```diff
@@ -28,6 +28,9 @@
   /** Handles the receipt of a single message. */
   public abstract void handle(T message) throws Exception;
 
+  /** Invoked when the channel this MessageHandler is on is active. */
+  public abstract void channelActive();
```

Comment:
also channelActive vs channelUnregistered is somewhat confusing. Either channelActive vs channelInactive, or channelRegistered vs channelUnregistered, or channelConnected vs channelDisconnected.